packages/editor-ui/src/router.ts

```diff
@@ -48,5 +48,14 @@ export default new Router({
 			path: '/',
 			redirect: '/workflow',
 		},
+		{
+			path: '/workflows/template/:id',
```

Comment:
can we change this to follow the convention we picked?

```suggestion
			path: '/workflows/templates/:id',
```